### DISCUSSION TITLE
Illustration clarification

### DISCUSSION THREAD

**Discussion by robyn-bautista** [twilio-labs/paste#3109] [Request: Illustration] [answered]
Hi! We've been using these illustrations in our comms canvas: 

I do not notice the rocket illustration in the [brand illustration library](https://www.figma.com/file/XeA9WNBcoojrdJbiz5Nj5q/Brand-Illustrations?node-id=31%3A4&t=QVMHbgNAius88AAt-1). So my question is do we have that rocket illustration as a component anywhere or is there another that's suggested to use instead?

**Comment by SiTaggart** [comment, marked as answer]
Hi there

The Brand team owns illustrations and so up until now we've taken the decision to not create any official components for these.

We've tended to help in a consultancy role between the product team wanting to implement, and the brand team to make sure the illustration is provided in the right format for the product team to consume.

We've never had an illustrator or anything on staff in Product UX, so we don't have the resources or capacity to do it the right way.

We'd love to create an asset component library for spot illustrations and brand moments in the future, it's just not high on the priority list right now unfortunately.

Reach out to the Brand team in Slack and see if they can help, also lean on the wider UX team in team-UX slack channel to see if someone has it in Figma already.

Thanks

**Reply by robyn-bautista** [reply]
Thanks Simon! I'll follow up in those channels
